### PR TITLE
ReadMe: Added Code of Conduct Section

### PR DESCRIPTION
**Pull request by anushkajain6459**
##Code of Conduct Section
Closes: [#94 ]

### CODE REVIEW

File: ReadMe.md [hunk 1]
```diff
@@ -146,6 +146,46 @@ We are primarily considering JavaScript (sorry, not TypeScript) as the base lang
...
+
+### Scope
+
+This Code of Conduct applies both within project spaces and in public spaces when an individual is representing the project or its community. Examples of representing a project or community include using an official project e-mail address, posting via an official social media account, or acting as an appointed a representative at an online or offline event. Representation of a project may be further defined and clarified by project maintainers.
+
+### Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting the project team at their [discord](https://discord.com/channels/731001496417730611/790974003011518474). All complaints will be reviewed and investigated and will result in a response that is deemed necessary and appropriate to the circumstances. The project team is obligated to maintain confidentiality with regard to the reporter of an incident. Further details of specific enforcement policies may be posted separately.
```
Comment (praveenscience) [inline ReadMe.md:185]: Update the Discord Link to our channel.

File: ReadMe.md [hunk 2]
```diff
@@ -146,6 +146,46 @@ We are primarily considering JavaScript (sorry, not TypeScript) as the base lang
 * Aditya Pahilwani ([LinkedIn](https://www.linkedin.com/in/aditya-pahilwani-93b42bb5/))
 * Tejan Singh ([LinkedIn](https://www.linkedin.com/in/tejansingh/))
 
+## Contributor Code of Conduct
+
+### Our Pledge
```
Comment (praveenscience) [inline ReadMe.md:151]: Can we have this in a separate file and link it from here? :) Thanks.

**Review by praveenscience — CHANGES_REQUESTED** [review]

File: ReadMe.md (hunk 2 above)
Comment (anushkajain6459) [reply ReadMe.md:151]: I actually didn't get what you are suggesting 😅 Are you suggesting that I should create a separate ReadMe file?

Comment (praveenscience) [reply ReadMe.md:151]: Separate Code of Conduct file. 😊  See the Contribution Guidelines section... 🔥

Comment (anushkajain6459) [reply ReadMe.md:151]: okay!! I will surely follow that.

File: ReadMe.md (hunk 1 above)
Comment (anushkajain6459) [reply ReadMe.md]: But the link to our channel won't be accessible by all. Shall I add it or not?

File: ReadMe.md (hunk 2 above)
Comment (anushkajain6459) [reply ReadMe.md]: Please check files changed section and any changes if required. 😀

File: ReadMe.md (hunk 1 above)
Comment (praveenscience) [reply ReadMe.md]: Yep, I know it. So that's completely fine.

File: ReadMe.md (hunk 2 above)
Comment (praveenscience) [reply ReadMe.md]: Will check now.

File: ReadMe.md (hunk 1 above)
Comment (anushkajain6459) [reply ReadMe.md]: Okay! 😀

Comment (anushkajain6459) [reply ReadMe.md]: Added the link of Devternship's discord channel. Please check.

**Review by praveenscience — CHANGES_REQUESTED** [review]
Cool, thanks. Can you link to this file from the main ReadMe please?

**Comment by anushkajain6459** [comment]
Linked Code_of_Conduct.md to main ReadMe. Please check

File: ReadMe.md [hunk 3]
```diff
@@ -77,6 +77,9 @@ Yes, we’re a part of GSSoC ’21. Look out for the Live Streams to get your is
 
 Please read the [Contribution Guidelines](./Contributing.md) of this project.
 
+### Code of Conduct
+Please read the [Code of Conduct](./Code_of_Conduct.md) of this project.
```
Comment (praveenscience) [inline ReadMe.md:81]: Add a single line space after the heading, please.

**Review by praveenscience — CHANGES_REQUESTED** [review]

File: ReadMe.md (hunk 3 above)
Comment (anushkajain6459) [reply ReadMe.md:81]: okay!

**Review by praveenscience — APPROVED** [review]
👍

**Pull request merged**